### PR TITLE
Adds steps to contributing guide for running some jobs locally

### LINKED ISSUE #172: [TRI-660] Add steps to the contributing guide on getting some jobs running locally
It would be really helpful I think to include very explicit instructions for how to get some example jobs running locally for contributors to this project. Currently we have the `nextjs-example` example inside the repo but it's got WAY too many jobs in it to be useful, especially for first time contributors.

So part of this issue would be to create a barebones nextjs example (maybe something called `examples/jobs-starter` or similar) that just has a few simple (scheduled and event triggers) jobs and then add instructions to the contributing guide for getting those into the local instance and running them

<sub>[TRI-660](https://linear.app/triggerdotdev/issue/TRI-660/add-steps-to-the-contributing-guide-on-getting-some-jobs-running)</sub>

### PR DESCRIPTION
**Pull request by tobihans**
This PR closes #172.

It adds the sample project with event and scheduled jobs. And documents the process inside the contribution guide.

/claim #172.

### CODE REVIEW

File: examples/jobs-starter/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "scripts": {
+    "dev": "concurrently --kill-others npm:dev:*",
+    "dev:next": "next dev",
+    "dev:trigger": "npx @trigger.dev/cli dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@trigger.dev/nextjs": "^1.0.0-next.8",
```
Comment (ericallam) [inline examples/jobs-starter/package.json]: These should point to the workspace versions, similar to how the `nextjs-example` project does. This also means you'll need to update the `tsconfig.json` of this project to use a path reference to the workspace dependencies, like so:

https://github.com/triggerdotdev/trigger.dev/blob/ba03a9ed20aef58e129e99a7acd7f540262350dc/examples/nextjs-example/tsconfig.json#L19

File: examples/jobs-starter/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jobs-starter",
```
Comment (ericallam) [inline examples/jobs-starter/package.json]: this should be `@examples/jobs-starter`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -84,6 +84,28 @@ branch are tagged into a release monthly.
...
+1. `cd` into `examples/jobs-starter`
+2. Create a `.env.local` file with the following content, 
+   replacing `[TRIGGER_DEV_API_KEY]` with an actual key:
+
+   ```
+   TRIGGER_API_KEY=[TRIGGER_DEV_API_KEY]
+   TRIGGER_API_URL=http://localhost:3030
+   ```
+
+3. Run the app:
```
Comment (ericallam) [inline CONTRIBUTING.md]: Is this meant to run just the jobs-starter app? If so it would be great to make that clear.

**Review by ericallam — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (tobihans) [reply CONTRIBUTING.md]: Yes, it provides the endpoint of the Trigger.dev instance running locally. I'll add that. Thanks

**Review by ericallam — APPROVED** [review]

**Pull request merged**